packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionTabContent.tsx

```diff
@@ -0,0 +1,174 @@
+import { makeStyles } from '@material-ui/core';
+import { DetailsPanel } from 'components/common/DetailsPanel';
+import { makeNodeExecutionDynamicWorkflowQuery } from 'components/Workflow/workflowQueries';
+import { WorkflowGraph } from 'components/WorkflowGraph/WorkflowGraph';
+import { TaskExecutionPhase } from 'models/Execution/enums';
+import { NodeExecutionIdentifier } from 'models/Execution/types';
+import { startNodeId, endNodeId } from 'models/Node/constants';
+import { Admin } from 'flyteidl';
+import * as React from 'react';
+import { transformerWorkflowToDag } from 'components/WorkflowGraph/transformerWorkflowToDag';
+import { checkForDynamicExecutions } from 'components/common/utils';
+import { dNode } from 'models/Graph/types';
+import { useContext, useEffect, useMemo, useState } from 'react';
+import { useQuery } from 'react-query';
+import { useNodeExecutionContext } from '../contextProvider/NodeExecutionDetails';
+import { NodeExecutionsByIdContext } from '../contexts';
+import { NodeExecutionsTable } from '../Tables/NodeExecutionsTable';
+import { tabs } from './constants';
+import { NodeExecutionDetailsPanelContent } from './NodeExecutionDetailsPanelContent';
+import { NodeExecutionsTimelineContext } from './Timeline/context';
+import { ExecutionTimeline } from './Timeline/ExecutionTimeline';
+import { ExecutionTimelineFooter } from './Timeline/ExecutionTimelineFooter';
+import { convertToPlainNodes, TimeZone } from './Timeline/helpers';
+
+export interface ExecutionTabContentProps {
+  tabType: string;
+  abortMetadata?: Admin.IAbortMetadata;
+}
+
+const useStyles = makeStyles(() => ({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: '1 1 100%',
+  },
+  container: {
+    display: 'flex',
+    flex: '1 1 0',
+    overflowY: 'auto',
+  },
+}));
+
+export const ExecutionTabContent: React.FC<ExecutionTabContentProps> = ({
+  tabType,
+  abortMetadata,
+}) => {
+  const styles = useStyles();
+  const { compiledWorkflowClosure } = useNodeExecutionContext();
+  const { staticExecutionIdsMap } = compiledWorkflowClosure
```

Comment:
you can add `dag` and `error` to the destructure list to cover the WorkflowGraph - right now it's still doing this exact call in that component